src/formatter.js

```diff
@@ -90,14 +91,16 @@ format.formatValue = function (value) {
   return value;
 };
 
-format.timestampFactory = function (aggregation) {
-  return function (timestamp) {
+format.timestampFactory = function (aggregation, offset) {
+  return function (timestamp, localTimezone) {
```

Comment:
The original intent with this returned function was to provide a common way to `chart.js` to call the formatter, whether it was numeric or timestamp.

This comment continues at `chart.js`